apiserver/facades/controller/caasapplicationprovisioner/provisioner.go

```diff
@@ -367,7 +379,7 @@ func (a *API) provisioningInfo(appName names.ApplicationTag) (*params.CAASApplic
 		Devices:              devices,
 		Constraints:          mergedCons,
 		Base:                 params.Base{Name: base.OS, Channel: base.Channel},
-		ImageRepo:            params.NewDockerImageInfo(imageRepoDetails, imagePath),
+		ImageRepo:            params.NewDockerImageInfo(imageRepoDetails, modelImage),
```

Comment:
We still want to use an image path based on the actual model agent version `vers`. The model agent version is the source of truth. The model operator image may not yet have been updated to reflect the model version. We only want to re-use the repo part from the model operator. Same in any other places.